ycms/cms/templates/bed_assignment/assign_bed_assignment.html

```diff
@@ -16,7 +16,8 @@ <h2 class="text-2xl font-semibold">
                                 {% translate "Assigning patient" %} {{ patient.first_name }}
                                 {{ patient.last_name }}
                             </h2>
-                            <a href="{% url 'cms:protected:ward_detail' pk=ward.id %}">
+                            {% url 'cms:protected:ward_detail' pk=ward.id as url %}
+                            <a href="{{ url }}">
```

Comment:
I think this can be undone du to the new JS, right?
```suggestion
                            <a href="{% url 'cms:protected:ward_detail' pk=ward.id %}">
```